scripts/cli-test.sh

```diff
@@ -7,14 +7,6 @@ export RUST_LOG=info
 export PEG_IN_AMOUNT=10000
 source ./scripts/setup-tests.sh
 
-# Test config en/decryption tool
-$FM_DISTRIBUTEDGEN config-decrypt --in-file $FM_CFG_DIR/server-0/config --out-file $FM_CFG_DIR/server-0/config-plaintext.json --password pass0
```

Comment:
@elsirion did you want to keep this tool around?

Reply:
Ideally yes for debugging, but not as urgent anymore so no big deal if you removed it.

Reply:
I think the test should still work with the secret cfg, right? If so, can you activate it again?